master/internal/api_trials_intg_test.go

```diff
@@ -712,12 +712,12 @@ func TestTrialAuthZ(t *testing.T) {
 			})
 			return err
 		}, true},
-		{"CanGetExperimentArtifacts", func(id int) error {
-			_, err := api.CompareTrials(ctx, &apiv1.CompareTrialsRequest{
-				TrialIds: []int32{int32(id)},
-			})
-			return err
-		}, false},
+		// {"CanGetExperimentArtifacts", func(id int) error {
```

Comment:
is this an intended change? if not, please uncomment

Reply:
Sorry there was an error with this that I forgot to fix. Done now.